CMakeLists.txt

```diff
@@ -18,10 +18,14 @@ ExternalProject_add(build-clang-format
   USES_TERMINAL_DOWNLOAD 1
   USES_TERMINAL_CONFIGURE 1
   USES_TERMINAL_BUILD 1
-  CMAKE_ARGS -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DLLVM_ENABLE_ZLIB=OFF -DLLVM_ENABLE_ZSTD=OFF -DLLVM_ENABLE_PROJECTS=clang
-  BUILD_COMMAND ${CMAKE_COMMAND} --build . --target clang-format
+  CMAKE_ARGS -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DLLVM_ENABLE_ZLIB=OFF -DLLVM_ENABLE_ZSTD=OFF -DLLVM_ENABLE_PROJECTS=clang -DLLVM_TARGETS_TO_BUILD=
+  BUILD_COMMAND ${CMAKE_COMMAND} --build . --target clang-format --config Release
 )
-set(clang-format-executable ${CMAKE_BINARY_DIR}/llvm/bin/clang-format${CMAKE_EXECUTABLE_SUFFIX})
+set(config-subfolder "")
+if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+  set(config-subfolder "Release")
+endif()
+set(clang-format-executable ${CMAKE_BINARY_DIR}/llvm/${config-subfolder}/bin/clang-format${CMAKE_EXECUTABLE_SUFFIX})
```

Comment:
Do we have any idea why this worked before? Is it a `scikit-build` vs. `-core` difference in default CMake arguments?

Reply:
Looking into this again I think the difference is that scikit-build was using Ninja on windows, and scikit-build-core is using MSVC. I'll try to fix this as ninja is faster and then we don't need this `config-subfolder` cruft.